### PR TITLE
Refresh environment library field in Lookdev

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
Fix for https://fogbugz.unity3d.com/f/cases/1262990/ 

The UI element showing the library name was not refreshed upon refreshing the library display + the library display is not refreshed on repaint, meaning that it wouldn't necessarily reflect changes done via code, now it should. 

**What did I test**: the repro case in various flavor. Directly as stated, after changing the library from lookdev and after the lookdev was open already.

### CODE REVIEW

**Review by iM0ve — CHANGES_REQUESTED** [review]
**Tested:**
The repro steps for the fb

**Issues:**
Change seems to break lookdev on my end. I checked same steps on HDRP/staging and there were no errors

Capture

lookdev

**Comment by FrancescoC-unity** [comment]
> **Tested:**
> The repro steps for the fb
> 
> **Issues:**
> Change seems to break lookdev on my end. I checked same steps on HDRP/staging and there were no errors
> 
> Capture
> 
> lookdev

Very odd :/ Will delete library and try on fresh project.

**Comment by FrancescoC-unity** [comment]
@iM0ve  Should be fixed it now

**Review by iM0ve — APPROVED** [review]
Seems to be working properly now

lookdev2

File: com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs [hunk 1]
```diff
@@ -533,7 +533,11 @@ void IViewDisplayer.SetTexture(ViewCompositionIndex index, Texture texture)
             }
         }
 
-        void IViewDisplayer.Repaint() => Repaint();
+        void IViewDisplayer.Repaint()
+        {
+            RefreshLibraryDisplay();
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs:538]: Repaint already call this. It seams not needed.

**Review by RSlysz — CHANGES_REQUESTED** [review]
Remove duplicate call to RefreshLibraryDisplay and it will be good

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs (hunk 1 above)
Comment (FrancescoC-unity) [reply com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs]: Done!

**Comment by sebastienlagarde** [comment]
issue adressed. merging

**Pull request merged**